### DISCUSSION TITLE
T3 Styled-Components TS Example

### DISCUSSION THREAD

**Discussion by nilssonj-tgc** [ben-rogerson/twin.macro#803] [Ideas]
Is there support for styled-components and typescript using t3? I see an example with emotion but not for styled-components.

I appreciate your work! Thanks :)

**Comment by ben-rogerson** [comment]
Yeah there should be support, I just haven't had the time to create a separate example for styled-components yet.
I'd like to release an example at some stage, but quite busy with a new job at the moment.

**Comment by ben-rogerson** [comment]
Here's [the styled-components example ](https://github.com/ben-rogerson/twin.examples/tree/master/t3-styled-components-typescript)
